### DISCUSSION TITLE
Why does this default stone generator not work?

### DISCUSSION THREAD

**Discussion by DemonSlayer112** [null2264/CobbleGen#12] [Q&A] [answered]
Im confused as why this doesnt work:
```
	"stoneGen": [
		{
			"id": "minecraft:stone",
			"weight": 100.0,
			"minY": 0
		},
		{
			"id": "minecraft:deepslate",
			"weight": 75.0,
			"minY": -64,
			"maxY": -1
		},
		{
			"id": "minecraft:tuff",
			"weight": 25.0,
			"minY": -64,
			"maxY": -1
		}
	]
```

**Comment by null2264** [comment, marked as answer]
Can you elaborate on what doesn't work?

Answer: After investigating further, the problem is actually maxY and minY value is flipped, maxY value should be in minY and vice versa. A better more straight forward range config is planned.

Currently proposed range config design:
```json5
{
    "range": "[-64, -1)"  // This would mean -64 up to (but not including) -1
    "range": "[-64, -1]"  // This would mean -64 up to (including) -1
    "range": "(-64, -1)"  // This would mean -63 up to (but not including) -1
}
```

**Reply by DemonSlayer112** [reply]
> Can you elaborate on what doesn't work?

The code I guess?
When I relog the client it doesn't do anything, only making stone regardless of the y level.

**Reply by null2264** [reply]
Interesting, it definitely works for me...

![image](https://user-images.githubusercontent.com/46041660/226506579-b2b66d63-7a31-40b2-8ef2-ecdd0e3ebd15.png)

Have you tried running `/reload`?

**Reply by null2264** [reply]
Also, make sure that the file is saved properly and your generated block is being generated BELOW y=-1

**Reply by DemonSlayer112** [reply]
> Also, make sure that the file is saved properly and your generated block is being generated BELOW y=-1

This sure is odd, so far it only works for deepslate cobblestone and not stone...
Ok! I removed the example custom stoneGen and the default one started working! :D
Idk why the custom would do this tho, it required a bedrock block i thought?

**Reply by null2264** [reply]
It should be, from my testing, there are no instances where it would generate without a modifier. Since it's working now, I'll mark this as answer. As for the custom stoneGen bug, can you provide me with your bugged config file? for when I have the time to look into it deeper.

**Reply by DemonSlayer112** [reply]
![image](https://user-images.githubusercontent.com/70122500/226701827-a8cb1ea6-8c45-41f5-8031-6da97bd848fb.png)
Alright this may be the issue here.
Turns out it does work but not at y -1, which is sort of odd?

Here's the config: [cobblegen.zip](https://github.com/null2264/CobbleGen/files/11032216/cobblegen.zip)

Figured stone having minY: 0 would make sure this doesnt happen?
And cobbled deepslate seems to work just fine for this.

**Reply by null2264** [reply]
Yeah, minY is not inclusive. I'll add an option for inclusive to avoid future confusion.
